README.md

```diff
@@ -6,8 +6,8 @@ A Bitcoin Gold docker image.
 
 ## Tags
 
-- `0.15.0.2-alpine`, `0.15-alpine`, `alpine`, `latest` ([0.15/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-gold/blob/master/0.15/alpine/Dockerfile))
-- `0.15.0.2`, `0.15`  ([0.15/Dockerfile](https://github.com/uphold/docker-bitcoin-gold/blob/master/0.15/Dockerfile))
+- `0.15.1-alpine`, `0.15-alpine`, `alpine`, `latest` ([0.15/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-gold/blob/master/0.15/alpine/Dockerfile))
```

Comment:
Since this is a `rc1` version, I would rather generate the tags `0.15.1-rc1` and `0.15.1-rc1-alpine` instead using the reserved tag for the final version `0.15.1`.